src/core/src/engine.rs

```diff
@@ -73,7 +73,11 @@ impl Engine for Poet2Engine {
 
         let (poet_pub_key, enclave_quote) = service.enclave.get_signup_parameters();
 
-        let mut wait_time =  Duration::from_secs(service.get_wait_time(chain_head.clone(), &validator_id, &poet_pub_key));
+        debug!("Signup info parameters: poet_pub_key = {}, enclave_quote = {}",
```

Comment:
Any reason why we're adding this debug message again after I removed in an earlier PR?